cli/dev/DevServer.ts

```diff
@@ -191,6 +194,43 @@ circuit.add(<MyCircuit />)
       },
     })
   }
+  private async handleExport(
+    ev: Pick<FileUpdatedEvent, "event_id" | "event_type"> & {
+      exportType: ExportFormat
+    },
+  ) {
+    const postEvent = async (
+      event: "FAILED_TO_EXPORT" | "EXPORT_CREATED",
+      content: Record<string, any>,
+    ) => {
+      this.fsKy.post("api/events/create", {
+        json: { event_type: event, ...content },
+        throwHttpErrors: false,
+      })
+    }
+
+    await exportSnippet({
+      filePath: this.componentFilePath,
+      format: ev.exportType,
+      browserDownload: true,
+      onExit: () => {},
+      onError: (e) => {
+        console.error("Failed to export: ", e)
+        postEvent("FAILED_TO_EXPORT", { message: e })
+      },
+      onSuccess: (response: {
+        fileName: string
+        mimeType: string
+        binaryData: Buffer
+      }) => {
+        postEvent("EXPORT_CREATED", {
+          fileName: response.fileName,
+          mimeType: response.mimeType,
+          binaryData: response.binaryData.toString("base64"),
```

Comment:
we don't want to use `binaryData`, it will crowd the event bus and cause issues. Create a file in the vfs via `/api/files/create` with a path we can ignore when building circuits e.g. `exports/gerbers.zip` then add a `download_url` or `download_path` to the event

Reply:
RunFrame will need to know to ignore all `exports/*` paths